librocksdb-sys/build_version.cc

```diff
@@ -3,13 +3,33 @@
 #include <memory>
 
 #include "rocksdb/version.h"
+#include "rocksdb/utilities/object_registry.h"
```

Comment:
The build_verion.cc is generated by building rocksdb.  The file is generated from build_version.cc.in.

Since https://github.com/facebook/rocksdb/pull/7949 updates build_version.cc.in, the resulting build_verion.cc will also reflect the change.

Reply:
Without updating our build_version.cc, it will result in the following link error, which symbol is introduced in https://github.com/facebook/rocksdb/pull/7949:

```
  = note: Undefined symbols for architecture arm64:
            "rocksdb::ObjectRegistry::builtins_", referenced from:
                rocksdb::ObjectRegistry::ObjectRegistry(std::__1::shared_ptr<rocksdb::ObjectLibrary> const&) in liblibrocksdb_sys-d6e9cb3766a4628b.rlib(object_registry.o)
          ld: symbol(s) not found for architecture arm64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
```